io/src/ensenso_grabber.cpp

```diff
@@ -963,7 +965,7 @@ pcl::EnsensoGrabber::processGrabbing ()
       if (num_slots<sig_cb_ensenso_point_cloud> () > 0 || num_slots<sig_cb_ensenso_images> () > 0 || num_slots<sig_cb_ensenso_point_cloud_images> () > 0)
       {
         pcl::PointCloud<pcl::PointXYZ>::Ptr cloud (new pcl::PointCloud<pcl::PointXYZ>);
-        boost::shared_ptr<PairOfImages> images (new PairOfImages);
+        std::unique_ptr<PairOfImages> images (new PairOfImages);
```

Comment:
This one gets published through `point_cloud_images_signal_` on line 1057. That signal expects a Boost shared pointer and is a part of public API. So we need to hold this change back I'm afraid.